dev/sg/sg_ci.go

```diff
@@ -201,6 +202,11 @@ sg ci build --help
 		Usage:   "Preview the pipeline that would be run against the currently checked out branch",
 		Flags: []cli.Flag{
 			&ciBranchFlag,
+			&cli.StringFlag{
+				Name:  "format",
+				Usage: "Output format for the preview, (valid values are 'markdown', 'json' and 'yaml' which will both output the raw pipeline in that format",
```

Comment:
```suggestion
				Usage: "Output format for the preview (one of 'markdown', 'json', and 'yaml')",
```

I think further details can go in the command description